Makefile

```diff
@@ -97,7 +97,10 @@ KUBEBUILDER_ASSETS?="$(shell $(ENVTEST) --arch=$(ENVTEST_ARCH) use -i $(ENVTEST_
 test: $(LIBGIT2) install-envtest test-api check-deps ## Run tests
 	KUBEBUILDER_ASSETS=$(KUBEBUILDER_ASSETS) \
 	GIT_CONFIG_GLOBAL=/dev/null \
-	go test $(GO_STATIC_FLAGS) ./... -coverprofile cover.out
+	go test $(GO_STATIC_FLAGS) \
+	  ./... \
+	  $(GO_USER_FLAGS) \
```

Comment:
Looks similar to what can be done in kustomize-controller with `GO_TEST_ARGS` https://github.com/fluxcd/kustomize-controller/blob/bcfd42466cf2fd24f490d1c3eac01feb03238114/Makefile#L23 . Maybe that'd be more consistent?